packages/@o3r-training/tools/package.json

```diff
@@ -0,0 +1,139 @@
+    "ng": "yarn nx",
+    "build:cjs": "swc src -d dist/cjs -C module.type=commonjs -q --strip-leading-paths",
+    "build:esm2015": "swc src -d dist/esm2015 -C module.type=es6 -q --strip-leading-paths",
+    "build:esm2020": "tsc -b tsconfig.build.json --pretty",
+    "build": "yarn nx build training-tools",
+    "postbuild": "yarn cpy 'package.json' dist && patch-package-json-main",
+    "prepare:publish": "prepare-publish ./dist",
+    "build:source": "yarn build:esm2020 && yarn build:cjs && yarn build:esm2015",
+    "build:cli": "tsc -b tsconfig.cli.json --pretty && yarn generate-cjs-manifest"
+  },
+  "bin": {
+    "o3r-extract-folder-structure": "./dist/cli/extract-folder-structure/extract-folder-structure.js"
+  },
+  "exports": {
+    "./package.json": {
+      "default": "./package.json"
+    },
+    ".": {
+      "module": "./dist/src/public_api.js",
+      "esm2020": "./dist/src/public_api.js",
+      "esm2015": "./dist/esm2015/public_api.js",
+      "es2020": "./dist/cjs/public_api.js",
+      "default": "./dist/cjs/public_api.js",
+      "typings": "./dist/src/public_api.d.ts",
+      "import": "./dist/src/public_api.js",
+      "node": "./dist/cjs/public_api.js",
+      "require": "./dist/cjs/public_api.js"
+    }
+  },
+  "peerDependencies": {
+    "@o3r/telemetry": "workspace:^",
+    "@schematics/angular": "~18.2.0",
+    "@webcontainer/api": "1.3.0-internal.2",
+    "eslint": "^8.57.0",
+    "rxjs": "^7.8.1",
+    "type-fest": "^4.10.2",
+    "typescript": "~5.5.4"
+  },
+  "peerDependenciesMeta": {
+    "@angular-devkit/architect": {
+      "optional": true
+    },
+    "@angular-devkit/core": {
+      "optional": true
+    },
+    "@angular/cli": {
+      "optional": true
+    },
+    "@o3r/telemetry": {
+      "optional": true
+    },
+    "eslint": {
+      "optional": true
+    },
+    "type-fest": {
+      "optional": true
+    }
+  },
+  "dependencies": {
+    "commander": "^12.0.0",
+    "tslib": "^2.6.2"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~18.2.0",
+    "@angular-devkit/core": "~18.2.0",
+    "@angular-devkit/schematics": "~18.2.0",
+    "@angular-eslint/eslint-plugin": "~18.3.0",
+    "@angular/animations": "~18.2.0",
+    "@angular/cli": "~18.2.0",
+    "@angular/common": "~18.2.0",
+    "@angular/compiler": "~18.2.0",
+    "@angular/compiler-cli": "~18.2.0",
+    "@angular/core": "~18.2.0",
+    "@angular/platform-browser": "~18.2.0",
+    "@angular/platform-browser-dynamic": "~18.2.0",
+    "@babel/core": "~7.25.0",
+    "@babel/preset-typescript": "~7.24.0",
+    "@compodoc/compodoc": "^1.1.19",
+    "@nx/eslint": "~19.5.0",
+    "@nx/eslint-plugin": "~19.5.0",
+    "@nx/jest": "~19.5.0",
+    "@nx/js": "~19.5.0",
+    "@o3r/build-helpers": "workspace:^",
+    "@o3r/eslint-plugin": "workspace:^",
+    "@o3r/telemetry": "workspace:^",
+    "@schematics/angular": "~18.2.0",
+    "@stylistic/eslint-plugin-ts": "~2.4.0",
+    "@swc/cli": "~0.4.0",
+    "@swc/core": "~1.7.0",
```

Comment:
We may check which deps is really needed by this package

Reply:
We use swc to build this package